src/platform/plugins/shared/data/public/query/filter_manager/lib/map_filter.ts

```diff
@@ -75,7 +75,7 @@ export function mapFilter(filter: Filter) {
   mappedFilter.meta.params = mapped.params;
   mappedFilter.meta.disabled = Boolean(mappedFilter.meta.disabled);
   mappedFilter.meta.negate = Boolean(mappedFilter.meta.negate);
-  mappedFilter.meta.alias = mappedFilter.meta.alias || null;
+  mappedFilter.meta.alias = mappedFilter.meta.alias;
```

Comment:
Gets rid of the defaulting to `null` as we treat it the same as `undefined` and the `null` value is causing this bug due to its inconsistency with how the TRADE team writes prebuilt rule filters.